zombies.js

```diff
@@ -137,8 +187,17 @@
  * @param {Item/Weapon/Food} item   The item to discard.
  * @return {boolean} true/false     Whether player was able to remove item from pack.
  */
-
-
+  discardItem(item){
+    var a = this.getPack().indexOf(item);
+    if (a===0 || a===1 || a===2){
```

Comment:
Is there a better way to handle if the item is in the pack, something where if the pack size was to increase, we wouldn't have to keep adding more expressions to this conditional?